app/views/submissions/_submission_responses.html.erb

```diff
@@ -5,7 +5,7 @@
   <% r = response %>
   <% question = r.custom_form_question %>
 
-  <% if params && YAML.load(params[:admin].to_s) %>
+  <% if context.can_admin? %>
```

Comment:
I believe this was the intention of the existing code?

Reply:
Either way, I appreciate getting the conditionals moved out of a view